addon/index.js

```diff
@@ -103,12 +150,20 @@ let RouterScrollMixin = Mixin.create({
     }
 
     const delayScrollTop = get(this, 'service.delayScrollTop');
+    const whenPainted = get(this, 'service.whenPainted');
```

Comment:
At the expense of being wordier, I'm wondering if these new properties could be more explicit.  Like, `scrollWhenPainted` and `scrollWhenIdle` or something like that?  Also -- would `settled` be a more accurate description than `idle`?

Reply:
great idea!  changed